static/updatePanel.sh

```
@@ -120,7 +120,19 @@ if [ "$delete_confirm" != "y" ]; then
   exit 1
 fi
 
-find "$install_dir" -mindepth 1 -maxdepth 1 ! -name 'backup' ! -name 'plugins' ! -name 'panel.tar.gz' -exec rm -rf {} +
+echo "Enabling maintenance mode"
+(cd "$install_dir" && php artisan down)
+if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)